controllers/clustercache/cluster_accessor.go

```diff
@@ -241,6 +241,7 @@ func (ca *clusterAccessor) Connect(ctx context.Context) (retErr error) {
 
 	if ca.Connected(ctx) {
 		log.V(6).Info("Skipping connect, already connected")
+		connectionUp.WithLabelValues(ca.cluster.String()).Set(1)
```

Comment:
I think we don't need this one here. The only way to get to Connected is through Connect and we're already setting the metric to 1 in l.269